sdcm/cluster.py

```diff
@@ -3437,10 +3437,14 @@ def __init__(self, params):
     @property
     def gemini_version(self):
         if not self._gemini_version:
-            result = self.nodes[0].remoter.run('cd $HOME; ./gemini --version')
-            # result : gemini version 1.0.1, commit ef7c6f422c78ef6b84a6f3bccf52ea9ec846bba0, date 2019-05-16T09:56:16Z
-            # take only version number - 1.0.1
-            self._gemini_version = result.stdout.split(',')[0].split(' ')[2]
+            try:
+                result = self.nodes[0].remoter.run('cd $HOME; ./gemini --version', ignore_status=True)
```

Comment:
create version regex in `sdcm/utils/version_utils.py`

Reply:
Done